arangod/Aql/ModificationOptions.cpp

```diff
@@ -60,5 +60,6 @@ void ModificationOptions::toVelocyPack(VPackBuilder& builder) const {
   builder.add("consultAqlWriteFilter", VPackValue(consultAqlWriteFilter));
   builder.add("exclusive", VPackValue(exclusive));
   builder.add("overwrite", VPackValue(overwrite));
+  builder.add("overwriteModeUpdate", VPackValue(overwriteModeUpdate));
```

Comment:
The value of `overwriteModeUpdate` is stringified here, but never read back from VelocyPack...
The deserialization of the attribute is missing in `ModificationOptions::ModificationOptions(VPackSlice const& slice)`.
Thus, it will not work in the cluster.